frontend/src/components/assessments/assessment-creation/QuestionCard.tsx

```diff
@@ -2,14 +2,12 @@ import * as React from "react";
 import {
   Box,
   Container,
-  Flex,
```

Comment:
the cards look a bit big imo - not sure if that's intended design

Reply:
i think i had to make them a bit bigger than they originally were in order to fit the `AssesssmentQuestions` section according to the specced design. i figured for those sort of design details we can ask them bout it once we ship the epic and they do product testing? otherwise lmk and i can resize